packages/test-scenarios/__tests__/scenario-test.js

```diff
@@ -155,6 +197,7 @@ Scenarios.fromProject(baseApp)
   })
   .forEachScenario((scenario) => {
     describe(scenario.name, () => {
+      const numberOfTests = scenario.name.includes('classicInRepoAddon-app') ? '7' : '5';
```

Comment:
I'd probably try to avoid doing this type of thing vs. passing in extra data into the tests. If you do this, it's going to quickly become a mess if you're having to support multiple variations. The tests will become a conditional hellscape, and will be extremely hard to read. Instead, you should determine if there's a way to pass additional data to the scenario by attaching that data from inside the scenario itself.

Reply:
I'll work on figuring this out. We expect slightly different file path results for classic vs in-repo. And for in-repo scenarios, we have a few more tests because we're testing for multiple addons case, e.g. 13 tests vs 5.

Reply:
So maybe we just need to split things out into multiple scenario iterations?

Reply:
Addressed by passing test info from each scenario and merging just those test files.